pom.xml

```diff
@@ -78,7 +79,12 @@
 				<version>${appuiVersion}</version>
 				<scope>provided</scope>
 			</dependency>
-
+			<dependency>
+				<groupId>org.openmrs.module</groupId>
+				<artifactId>appframework-api</artifactId>
+				<version>${appframeworkVersion}</version>
+				<scope>provided</scope>
```

Comment:
If this is for just tests, shouldn't you change the scope?

Reply:
Changed to test @dkayiwa